python/lsst/ts/vent/controller/controller.py

```diff
@@ -231,8 +291,9 @@ async def last8faults(self) -> list[tuple[int, str]]:
         """
 
         self.log.debug("last8faults")
+        assert self.connected
         rvals = await self.vfd_client.read_holding_registers(
-            slave=1, address=vf_drive.Registers.FAULT_REGISTER, count=8
+            slave=self.cfg.slave, address=vf_drive.Registers.FAULT_REGISTER, count=8
```

Comment:
I would avoid the use of terms like `slave`. Please, consider renaming this; e.g. client or any other less offensive term.